bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/resources/ESH-INF/thing/daikinmadoka.xml

```diff
@@ -97,7 +106,14 @@
 	<channel-type id="brc1h_homekitTargetHeatingCoolingMode">
 		<item-type>String</item-type>
 		<label>HomeKit Target Mode</label>
-		<description>Valid values: Off, CoolOn, CoolOff, Auto</description>
+		<command>
+            <options>
```

Comment:
please reapply the formatter - here are spaces instead of tabs, which breaks the indentation

Reply:
I don't undestand what's wrong with my Eclipse. I select all, source, format, it appears good in eclipse XML editor, in VI - it is still weird. Have to modify it manually with VI ... Even Atom see it well.